sdgx/data_models/combiner.py

```diff
@@ -35,6 +35,10 @@ class MetadataCombiner(BaseModel):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        if kwargs["relationships"]:
+            self.relationships = kwargs["relationships"]
+        else:
+            self.relationships = []
```

Comment:
This would be better
```suggestion
    def __init__(self, relationships=None, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.relationships = relationships or []
```